bench/workgen/workgen.cxx

```diff
@@ -2324,6 +2583,24 @@ WorkloadOptions::WorkloadOptions()
       "timestamp forward");
     _options.add_bool("max_idle_table_cycle_fatal", max_idle_table_cycle_fatal,
       "print warning (false) or abort (true) of max_idle_table_cycle failure");
+    _options.add_int("dynamic_create_period", dynamic_create_period,
+      "if greater than 0, creates tables every this many seconds. The number of tables created "
+      "is specified by the dynamic_create_count setting");
+    _options.add_int("dynamic_create_count", dynamic_create_count,
+      "create this many number of tables every dynamic_create_period seconds");
+    _options.add_int("dynamic_create_target", dynamic_create_target,
```

Comment:
I suggest switching the order of these two options then word the text as below. I'm not sure what is meant by "if not 0" so I've left it out:
```
   _options.add_int("dynamic_create_trigger", dynamic_create_trigger,
      "start creating tables when the database drops below this size in MB"
  _options.add_int("dynamic_create_target", dynamic_create_target,
     "stop creating tables when the database exceeds this size in MB"
```

Reply:
done.